coverage/clover.xml

```diff
@@ -1,6 +1,83 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<coverage generated="1584478565807" clover="3.2.0">
```

Comment:
I still think this file should be added to the .env.

Reply:
I recall you mentioning that, and you're right that it's a huge pain to deal with.
Removing it.